.gitignore

```diff
@@ -0,0 +1,2 @@
+venv/
```

Comment:
Please remove all the sphinx files. They are already being added in PR # 1 (from Akshay). If you have anything to add beyond what Akshay has added, let's discuss and then you can submit it as a separate PR.